html5up-story/index.html

```diff
@@ -0,0 +1,356 @@
+<!DOCTYPE HTML>
+<!--
+	Story by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+	<head>
+		<title>Priya Vaidya - Portfolio</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+	</head>
+	<body class="is-preload">
+
+		<!-- Wrapper -->
+			<div id="wrapper" class="divided">
+
+				<!-- One -->
+					<section class="banner style1 orient-left content-align-left image-position-right fullscreen onload-image-fade-in onload-content-fade-right">
+						<div class="content">
+							<h1>Priya Vaidya</h1>
+							<p class="major">I'm an aspiring Fullstack Software Developer in Los Angeles, CA. I am currently a student at <a href="https://lambdaschool.com/">Lambda School</a>. </p>
+							<ul class="actions stacked">
+								<li><a href="#first" class="button big wide smooth-scroll-middle">View My Work</a></li>
+							</ul>
+						</div>
+						<div class="image">
+							<img src="images/la.jpg" alt="Los Angeles" />
+						</div>
+					</section>
+
+				<!-- Two -->
+					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in" id="first">
+						<div class="content">
+							<h2>Responsive Design Project</h2>
+							<p>An adaptive version of the "Great Idea" design. It is a site that would be responsive for desktops, tablets and moblie devices.</p>
+							<ul class="actions stacked">
+								<li><a href="https://github.com/pvaidya56/responsive-web-design-I" target="_blank" class="button">View my Code</a></li>
+							</ul>
+						</div>
+						<div class="image.main">
+							<img src="images/collage.png" alt="" />

```

Comment:
The project looks good, Priya! Just wondering if you are going to make this div with the image responsive in the future. 

> <div class="image.main">
> 							<img src="images/collage.png" alt="" />
> </div>